dbmigration/20220527150154_payload_job_run_name_to_bigint.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+UPDATE release_job_runs SET name='0' WHERE name='.';
+ALTER TABLE release_job_runs ALTER COLUMN name TYPE bigint USING (name::bigint);
+ALTER TABLE release_job_runs RENAME COLUMN name to prow_job_run_id
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+ALTER TABLE release_job_runs RENAME COLUMN prow_job_run_id to name
+ALTER TABLE release_job_runs ALTER COLUMN name TYPE text USING (name::text);
+UPDATE release_job_runs SET name='.' WHERE name='0';
+-- +goose StatementEnd
```

Comment:
This was causing a problem with some queries that casting seems to fail on. Figured it was best to get the column correct.